src/utils/hooks/api/loans/use-get-account-positions.tsx

```diff
@@ -1,212 +1,45 @@
-    ({ amount, currency, ...rest }) => ({
-      // MEMO: compute total value based on collateral threshold (not full lend amount value)
-      amount: amount.mul(assets[currency.ticker].liquidationThreshold),
-      currency,
-      ...rest
-    })
-  );
-
-  const collateralAmountUSD = getPositionsSumOfFieldsInUSD(
-    'amount',
-    collateralPositionsWithAppliedCollateralThreshold,
-    prices
-  );
-  const liquidationAmountUSD = getPositionsSumOfFieldsInUSD(
-    'amount',
-    collateralPositionsWithAppliedLiquidationThreshold,
-    prices
-  );
-
-  const collateralizedAmountUSD = getPositionsSumOfFieldsInUSD('amount', collateralLendPositions, prices);
-  const earnedDeptAmountUSD = getPositionsSumOfFieldsInUSD('accumulatedDebt', borrowPositions, prices);
-
-  // TODO: This is temporary, atleast until earned interest
-  // is moved into squid.
-  // const totalEarnedRewardsUSDValue =
-  //   convertMonetaryAmountToValueInUSD(subsidyRewards, getTokenPrice(prices, subsidyRewards.currency.ticker)?.usd) || 0;
-  // const netAmountUSD = earnedInterestAmountUSD.add(totalEarnedRewardsUSDValue).sub(earnedDeptAmountUSD);
-
-  const netAPY = getNetAPY(lendPositions, borrowPositions, assets, supplyAmountUSD, prices);
-
-  return {
-    supplyAmountUSD,
-    borrowAmountUSD,
-    earnedInterestAmountUSD: new Big(0),
-    collateralAmountUSD,
-    liquidationAmountUSD,
-    collateralizedAmountUSD,
-    earnedDeptAmountUSD,
-    netAmountUSD: new Big(0),
-    netAPY
-  };
-};
-
 type UseGetAccountPositions = {
   data: Partial<AccountPositionsData> & {
-    statistics: AccountPositionsStatisticsData | undefined;
     hasCollateral: boolean;
   };
   refetch: () => void;
 };
 
 const useGetAccountPositions = (): UseGetAccountPositions => {
-  const accountId = useAccountId();
-
-  const prices = useGetPrices();
-  const { data: assets } = useGetLoanAssets();
+  const { data: lendPositions, refetch: lendPositionsRefetch } = useGetLendPositionsOfAccount();
 
-  const { data: positions, error: positionsError, refetch: refetchPositions } = useQuery({
-    queryKey: ['positions', accountId],
-    queryFn: () => accountId && getAccountPositionsQuery(accountId),
-    enabled: !!accountId,
-    refetchInterval: BLOCKTIME_REFETCH_INTERVAL
-  });
-
-  const { data: subsidyRewards } = useGetAccountSubsidyRewards();
-
-  // MEMO: we dont need assets as a dependency, since we only use the collateral threshold and
-  // it's value is very unlikely to change
-  const statistics = useMemo(() => {
-    if (!assets || !positions || !subsidyRewards || !prices) {
-      return undefined;
-    }
-
-    return getAccountPositionsStats(assets, positions.lendPositions, positions.borrowPositions, subsidyRewards, prices);
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [positions, prices, subsidyRewards]);
-
-  useErrorHandler(positionsError);
+  const { data: borrowPositions, refetch: borrowPositionsRefetch } = useGetBorrowPositionsOfAccount();
```

Comment:
`getAccountPositionsQuery` was simplified using `useGetBorrowPositionsOfAccount` and `useGetLendPositionsOfAccount`.